server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -561,7 +561,6 @@ public synchronized void close() throws IOException {
         toClose.add(() -> stopWatch.stop().start("transport"));
         toClose.add(injector.getInstance(TransportService.class));
         toClose.add(injector.getInstance(NodeMetrics.class));
-        toClose.add(injector.getInstance(IndicesService.class));
```

Comment:
Good catch. Thanks Yang! I meant to write `IndicesMetrics` but typed `IndicesServices` instead. Could you please update it to `IndicesMetrics`?

Reply:
Ah thanks for the explanation. Pushed 39c4087a2b8c28e134cf8b80cf770baf69951294
Also updated the PR title and description.